### ISSUE TITLE
The left information bar does not display

### ISSUE THREAD

**Issue by MOLI-max** [tukui-org/ElvUI#1350]
### Troubleshooting Steps

I enabled the left information bar in the settings, but after reloading the plugin, the left information bar no longer appears. The right information bar is normal and does not display even after logging in again. I need to enable it again in the settings to display it, and I need to do this every time I log in.
![微信截图_20240904235723](https://github.com/user-attachments/assets/e2656050-ac06-4c6c-af28-a369ff29c383)

### Describe the expected behavior and what actually happened?

It should display continuously after starting the left information bar, even if I log in or reload the plugin, it will still display normally, but it is not working properly now

### Lua Errors

```shell
Message: Interface/AddOns/ElvUI/Core/General/Math.lua:416: bad argument #1 to 'abs' (number expected, got table)
Time: Wed Sep  4 23:45:50 2024
Count: 1
Stack: Interface/AddOns/ElvUI/Core/General/Math.lua:416: bad argument #1 to 'abs' (number expected, got table)
[string "@Interface/AddOns/ElvUI/Core/General/Math.lua"]:416: in function `FormatMoney'
[string "@Interface/AddOns/ElvUI/Core/Modules/DataTexts/Gold.lua"]:84: in function <...terface/AddOns/ElvUI/Core/Modules/DataTexts/Gold.lua:56>
[string "@Interface/AddOns/ElvUI/Core/Modules/DataTexts/Gold.lua"]:157: in function `eventFunc'
[string "@Interface/AddOns/ElvUI/Core/Modules/DataTexts/DataTexts.lua"]:429: in function `AssignPanelToDataText'
[string "@Interface/AddOns/ElvUI/Core/Modules/DataTexts/DataTexts.lua"]:594: in function `UpdatePanelInfo'
[string "@Interface/AddOns/ElvUI/Core/Modules/DataTexts/DataTexts.lua"]:624: in function `LoadDataTexts'
[string "@Interface/AddOns/ElvUI/Core/Modules/DataTexts/DataTexts.lua"]:846: in function `?'
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:109: in function <...ies/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:19: in function <...ies/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@Interface/AddOns/ElvUI_Libraries/Core/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceEvent-3.0/AceEvent-3.0.lua"]:120: in function <...UI_Libraries/Core/Ace3/AceEvent-3.0/AceEvent-3.0.lua:119>
```

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/main/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [ ] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by MOLI-max** [comment]
![微信截图_20240904235723](https://github.com/user-attachments/assets/7b376ce3-4a2c-4061-897a-edb3706a3407)

**Comment by 1Botanica1** [comment]
Do you get the lua error with only ElvUI? 

Not sure which bar you are referring to.  If it is one that is provided by another addon, yes, you need the other addon enabled for it to show.

**Comment by MOLI-max** [comment]
> Do you get the lua error with only ElvUI?
> 
> Not sure which bar you are referring to. If it is one that is provided by another addon, yes, you need the other addon enabled for it to show.

Yes, only ElvUI. In the screenshot, there should have been an information bar displayed in the lower left corner, which is the same as the information bar in the lower right corner. However, now there is no information bar in the lower left corner. The activation of this information bar can be found in ElvUI Settings - Information Text - Panel - Left Information Bar

**Comment by Azilroka** [comment]
Somehow your table is broken for your characters gold.

**Comment by Repooc** [comment]
stale
